README.md

```diff
@@ -1,38 +1,114 @@
 Minecraft Server Control Script
-=============================
+===============================
 
 A powerful command-line control script for UNIX and Linux powered Minecraft servers.
 
-
-## Index
-* [Features](#features)
+# Index
+* [Overview](#overview)
+* [Prerequisites for installation](#prerequisites-for-installation)
+  * [Required programs](#required-programs)
+  * [Configuring the firewall / NAT](#configuring-the-firewall--nat)
+  * [Mapping software (optional)](#mapping-software-optional)
 * [Installation](#installation)
-* [Usage](#usage)
-* [Server Customization](#server-customization)
+  * [Downloading the script](#downloading-the-script)
+  * [Configuration](#configuration)
+* [Creating and importing worlds](#creating-and-importing-worlds)
+  * [Create new world](#create-new-world)
+  * [Import existing world](#import-existing-world)
+* [Adjusting server options](#adjusting-server-options)
+  * [The mscs.properties file](#the-mscs.properties-file)
+  * [The msctl file](#the-msctl-file)
+* [Automated backups and restarts](#automated-backups-and-restarts)
+  * [Scheduling backups](#scheduling-backups)
+  * [Removing backups after X days](#removing-backups-after-x-days)
+  * [Scheduling restarts](#scheduling-restarts)
+* [Mapping the world](#mapping-the-world)
+* [Command reference](#command-reference)
+  * [Examples](#examples)
 * [License](LICENSE)
 * [Issues](#issues)
 
+## Overview
+**M**inecraft **S**erver **C**ontrol **S**cript (**MSCS**) is a server-management script for UNIX and Linux powered Minecraft servers. 
+Features include:
 
-## Features
 * Run multiple Minecraft worlds.
 * Start, stop, and restart single or multiple worlds.
 * Create, delete, disable, and enable worlds.
 * Includes support for additional server types: [Forge](http://www.minecraftforge.net/), [BungeeCord](http://www.spigotmc.org/wiki/bungeecord/), [SpigotMC](http://www.spigotmc.org/wiki/spigot/), etc.
 * Users automatically notified of important server events.
-* Uses the Minecraft [Query protocol](http://wiki.vg/Query) to keep track of current server conditions.
+* Uses the Minecraft [Query protocol](http:b //wiki.vg/Query) to keep track of current server conditions.
 * LSB and systemd compatible init script, allows for seamless integration with your server's startup and shutdown sequences.
 * Map worlds using the [Minecraft Overviewer](http://overviewer.org/) mapping software.
-* Backup worlds, and remove backups older than X days.
+* Automatically backup worlds, remove backups older than X days, and restart worlds.
 * Update the server and client software automatically.
 * Send commands to a world server from the command line.
 
-See the [Usage](#usage) section below for a description on how to use these features.
+Most of the general/beginner documentation is located in this file. For additional documentation that is not located in this file, please check the [wiki](https://github.com/Roflicide/MinecraftServerControlScript/wiki). 
```

Comment:
Hi Joshua,
Thank you for taking the time to review my edits! I am new to both linux
and github, and if you have the time, I was wondering if you could answer a
few questions about the edits you proposed:

1. msctl file



> * You should not be editing this file [msctl]. There is a section in msctl
> that will load config from other locations. Locations include
> $HOME/mscs.conf $HOME/.config/mscs/mscs.conf, or /etc/default/mscs (mscs
> set this value and calls msctl)*
>
>
I don't quite understand why it's not a good idea to edit this file. I saw
that there were options for settings in this file, so why do I need to edit
a file elsewhere?
I also tried looking for the files above in all of the paths you listed
above but I couldn't seem to find any of the mscs.conf files. I also
modified this file directly and it seemed to work fine for my server.

2. location of msctl

>
> >
> > >
> > > In README.md
> <https://github.com/sandain/MinecraftServerControlScript/pull/111#discussion_r51283134>
> :
> > >
> > > > +### Scheduling backups
> > > > +MSCS uses [**cron**](https://en.wikipedia.org/wiki/Cron), a
> scheduler software that can run programs on a set interval of time. Below
> is an example of one way how you could setup backups via cron to backup a
> world every 2 hours:
> > > > +
> > > > +Type the following (in any directory):
> > > > +
> > > > +  ````
> > > > +  export EDITOR=vim
> > > > +  crontab -e
> > > > +  ````
> > > > +  Page down until you get to an empty line. Then paste the
> following:
> > > > +
> > > > +  ````
> > > > +  0 */2 * * *  PATH-TO-MSCS backup myWorld
> > > > +  ````
> > > > +  * `0 */2 * * *` is the time interval to backup. This particular
> expression means backup every 2 hours. You can change this to 3, 4, 5 or
> whatever amount of hours to backup X amount of hours. You can also backup
> according to days, minutes, seconds, the time of the day, and more. See
> [the wiki page](
> https://github.com/sandain/MinecraftServerControlScript/wiki/Backup-and-Restore)
> for more information.
> > > > +  * `PATH-TO-MSCS` is the path of where the `mscs` file is (which
> will be inside the `

 *This should point to where msctl is installed, if installed with the
> makefile, this would be /usr/local/bin/msctl *



I guess I'm confused on the difference between msctl and mscs. I thought
that, since the command was "mscs", that the mscs file would be the one I
wanted to point towards backups. Why do I need to point it towards the
msctl file? I pointed it towards the mscs file with my server and it's
working just fine.

3. git pull vs git clone

> The location where MSCS is downloaded from git should not be used.
>
> Ideally you would do a git pull, then sudo make install to upgrade the
> script.
>
Why should it not be used? I don't understand why git pull should be used
instead of git clone. I tried doing some research on the difference between
the two and found that
"

They're basically the same, except clone will setup additional remote
tracking branches, not just master" -
http://stackoverflow.com/questions/3620633/what-is-the-difference-between-pull-and-clone-in-git


From what I got from that was their essentially the same, so I don't
understand why the difference is important.


Thanks, and sorry for the plethora of questions,

Michael